bdai_ros2_wrappers/bdai_ros2_wrappers/node.py

```diff
@@ -87,3 +89,21 @@ def shutdown(self) -> None:
     # def __del__(self) -> None:
     #     """Destructor"""
     #     self.shutdown()
+
+
+class FriendlyNode(Node):
```

Comment:
Maybe "NonReentrantNode" or "AutoScalingNode"?  Friendly is a pretty awesome name but doesn't exactly make it obvious it goes with the other stuff :)

Reply:
Yeah, `FriendlyName` is a throwaway name :sweat_smile:

We could just call it `Node`. That way downstream code can switch from one to the other by changing the import statements ie. from `from rclpy.node import Node` to `from bdai_ros.node import Node`.